sys/timeb.h / ftime are legacy, so don't enable them for uClibc 
---
 apps/speed.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: openssl-0.9.8g/apps/speed.c
===================================================================
--- openssl-0.9.8g.orig/apps/speed.c
+++ openssl-0.9.8g/apps/speed.c
@@ -113,7 +113,7 @@
 #elif !defined(OPENSSL_SYS_MSDOS) && !defined(OPENSSL_SYS_VXWORKS) && (!defined(OPENSSL_SYS_VMS) || defined(__DECC))
 # define TIMES
 #endif
-#if !defined(_UNICOS) && !defined(__OpenBSD__) && !defined(sgi) && !defined(__FreeBSD__) && !(defined(__bsdi) || defined(__bsdi__)) && !defined(_AIX) && !defined(OPENSSL_SYS_MPE) && !defined(__NetBSD__) && !defined(OPENSSL_SYS_VXWORKS) /* FIXME */
+#if !defined(_UNICOS) && !defined(__OpenBSD__) && !defined(sgi) && !defined(__FreeBSD__) && !(defined(__bsdi) || defined(__bsdi__)) && !defined(_AIX) && !defined(OPENSSL_SYS_MPE) && !defined(__NetBSD__) && !defined(OPENSSL_SYS_VXWORKS) && !defined(__UCLIBC__) /* FIXME */
 # define TIMEB
 #endif
 
